app/help/en/gateways/ioxbank.md

```diff
@@ -0,0 +1,42 @@
+# ioxbank Instant Swap Service
+A Bridge Instant Swap Service is responsible for moving cryptocurrencies between BitShares and other Blockchains. You can easily identify those cryptocurrencies supported by ioxbank, they are prefixed with **IOB.** and marked as featured in dashboard page.
```

Comment:
please change wording to gateway, or remove bridge. Bridge in the context of the wallet is the additional service that blocktrades offers (swap ETH to BTS).